tests/test_move.py

```diff
@@ -37,6 +38,42 @@ def test_move_fs(self):
             self.assertEqual(dst_file1_info.modified, src_file1_info.modified)
             self.assertEqual(dst_file2_info.modified, src_file2_info.modified)
 
+    def test_move_file(self):
+        with open_fs("temp://") as temp:
+            syspath = temp.getsyspath("/")
+            a = open_fs(syspath)
+            a.makedir("dir")
+            b = open_fs(join(syspath, "dir"))
+            b.writetext("file.txt", "Content")
+            fs.move.move_file(b, "file.txt", a, "here.txt")
+            self.assertEqual(a.readtext("here.txt"), "Content")
+            self.assertFalse(b.exists("file.txt"))
+
+    def test_move_file_different_mems(self):
+        with open_fs("mem://") as src, open_fs("mem://") as dst:
+            src.writetext("source.txt", "Source")
+            fs.move.move_file(src, "source.txt", dst, "dest.txt")
+            self.assertFalse(src.exists("source.txt"))
+            self.assertEqual(dst.readtext("dest.txt"), "Source")
+
+    def test_move_file_fs_urls(self):
+        # create a temp dir to work on
+        with open_fs("temp://") as tmp:
+            path = tmp.getsyspath("/")
+
+            tmp.writetext("file.txt", "Content")
+            tmp.makedir("subdir")
+            fs.move.move_file(path, "file.txt", join(path, "subdir"), "file.txt")
```

Comment:
I think you need to pass in fs-urls, not syspaths? See https://docs.pyfilesystem.org/en/latest/reference/base.html#fs.base.FS.geturl

Reply:
Cleared that up 👍